### ISSUE TITLE
image function in darknet

### ISSUE THREAD

**Issue by AshleyRoth** [AlexeyAB/darknet#2322]
Hello. Could you please tell me. Darknet has its own image format "image", how can I use it for my own purposes? For example, I want to output it using cvShowImage, but using the following code it is impossible to do it. 

```
image im;
... (some code)
cvShowImage("windo", im);
cvWaitKey(0);
```
I know that there are already functions, but I want to understand exactly how to work with "image", like, for example, I can transfer my "image" to Ipimage

Regards, Ash.

**Comment by AlexeyAB** [comment]
Hi, use `show_image(im, "predictions");`

**Comment by AshleyRoth** [comment]
@AlexeyAB
Yes, i know about it. But I would like to understand how this "image" works, I want to try to capture the output image with the drawn boxes, I need to get the "ImageData" property from IpImage, but in "image" I don’t know how to do this. Therefore, I would like to understand how to use for their own purposes.
